src/main/java/hudson/util/VersionNumber.java

```diff
@@ -513,16 +513,26 @@ public boolean isNewerThan(VersionNumber rhs) {
         return compareTo(rhs) > 0;
     }
 
+    /**
+     * Returns a digit (numeric component) by its position
+     *
+     * @param idx Digit position we want to retrieve
```

Comment:
Does the index start at 0 or 1. According to the tests, 1. It should be noted in the documentation.